### PR TITLE
Add new keyboard AT3

### PR DESCRIPTION
**Pull request by anubhavd7**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/atset/at16/at16.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "quantum.h"
+
+#define LAYOUT( \
```
Comment (drashna) [inline keyboards/atset/at16/at16.h:17]:
```suggestion
#define LAYOUT_ortho_4x4( \
```

File: keyboards/atset/at16/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ * You should have received a copy of the GNU General Public License 
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+ */ 
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+
+[0] = LAYOUT(
```
Comment (drashna) [inline keyboards/atset/at16/keymaps/default/keymap.c:21]:
```suggestion
[0] = LAYOUT_ortho_4x4(
```

File: keyboards/atset/at16/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	
+[1] = LAYOUT(
+  KC_NO,   KC_NO,   KC_NO,  KC_NO,
+  KC_NO,   KC_NO,   KC_NO,  KC_NO,
+  KC_NO,   KC_NO,   KC_NO,  KC_NO,
+  KC_NO,   KC_NO,   KC_NO,  KC_NO
+),
+
+	
+[2] = LAYOUT(
```
Comment (drashna) [inline keyboards/atset/at16/keymaps/default/keymap.c:37]:
```suggestion
[2] = LAYOUT_ortho_4x4(
```

File: keyboards/atset/at16/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+ * 
+ * You should have received a copy of the GNU General Public License 
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+ */ 
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] =
+        LAYOUT(
```
Comment (drashna) [inline keyboards/atset/at16/keymaps/via/keymap.c:20]:
```suggestion
        LAYOUT_ortho_4x4(
```

File: keyboards/atset/at16/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+                KC_F10,  KC_F11,  KC_F12, KC_F5,	
+                KC_7,    KC_8,    KC_9,   KC_F1,
+                KC_4,    KC_5,    KC_6,   KC_F2,
+                KC_1,    KC_2,    KC_3,   KC_F3
+              ),
+    [1] =
+        LAYOUT(
```
Comment (drashna) [inline keyboards/atset/at16/keymaps/via/keymap.c:27]:
```suggestion
        LAYOUT_ortho_4x4(
```

File: keyboards/atset/at16/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+                KC_NO , KC_NO, KC_NO, KC_NO,
+                KC_NO , KC_NO, KC_NO, KC_NO,
+                KC_NO , KC_NO, KC_NO, KC_NO,
+                KC_NO , KC_NO, KC_NO, KC_NO
+              ),
+    [2] =
+        LAYOUT(
```
Comment (drashna) [inline keyboards/atset/at16/keymaps/via/keymap.c:34]:
```suggestion
        LAYOUT_ortho_4x4(
```

File: keyboards/atset/at16/keymaps/via/keymap.c [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+                KC_NO , KC_NO, KC_NO, KC_NO,
+                KC_NO , KC_NO, KC_NO, KC_NO,
+                KC_NO , KC_NO, KC_NO, KC_NO,
+                KC_NO , KC_NO, KC_NO, KC_NO
+              ),
+    [3] =
+        LAYOUT(
```
Comment (drashna) [inline keyboards/atset/at16/keymaps/via/keymap.c:41]:
```suggestion
        LAYOUT_ortho_4x4(
```

File: keyboards/atset/at16/readme.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+# ATSET AT16
+
+A special 3 key keyboard powered by QMK.
```
Comment (drashna) [inline keyboards/atset/at16/readme.md:3]: 3 key?

File: keyboards/atset/at16/rules.mk [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
```
Comment (drashna) [inline keyboards/atset/at16/rules.mk:18]:
```suggestion
AUDIO_ENABLE = no           # Audio output

LAYOUTS = ortho_4x4
```

File: keyboards/atset/at16/readme.md (hunk 8 above)
Comment (anubhavd7) [reply keyboards/atset/at16/readme.md]: sorry sir my mistake

**Comment by drashna** [comment]
also, just a heads up, adding more files means that it will take longer to get merged, as we have to re-review the PR when they're added.

**Comment by anubhavd7** [comment]
> also, just a heads up, adding more files means that it will take longer to get merged, as we have to re-review the PR when they're added.

ok sir

**Comment by anubhavd7** [comment]
sir can you please check my pull request.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**